CompetitionTemplate/src/competition/subsystems/vision/InertJetsonServer.java

```diff
@@ -0,0 +1,34 @@
+package competition.subsystems.vision;
+
+import java.util.function.Consumer;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+public class InertJetsonServer implements JetsonServer {
```

Comment:
JetsonServer should probably have some sort of isConnectionHealthy() method that this class can always return FALSE on (and the real class could figure that out in a better way)